var/spack/repos/builtin/packages/fuse-overlayfs/package.py

```diff
@@ -15,6 +15,7 @@ class FuseOverlayfs(AutotoolsPackage):
 
     license("GPL-2.0-or-later")
 
+    version("1.13", sha256="96d10344921d5796bcba7a38580ae14a53c4e60399bb90b238ac5a10b3bb65b2")
```

Comment:
I thought about that, but I don't think virtuals and underlying providers will work with version restrictions like that as is. Might be wrong, it's been a while since I've dug in to it.

Reply:
Maybe this?
```
    depends_on("libfuse@3.2.1:", when="^libfuse")
```